modules/nf-core/rseqc/inferexperiment/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process RSEQC_INFEREXPERIMENT"
+    script "../main.nf"
+    process "RSEQC_INFEREXPERIMENT"
+    config "./nextflow.config"
+    
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rseqc"
+    tag "rseqc/inferexperiment"
+
+    test("sarscov2 - [[meta] - bam] - bed") {
+
+        when {
+            params {
+                // define parameters here. Example:
+                // outdir = "tests/results"
+            }
+            process {
+                """
+                input[0] = [ 
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
+                ]
+    	        input[1] = file(params.test_data['sarscov2']['genome']['test_bed'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assert process.success
```

Comment:
assertAll()

Reply:
This is literally what nf-test creates- could that be changed if you don't like this?

Reply:
I'm actually fine with this for now as the test is working.
We should lint for that, but let's merge that one, and do another pass in all modules to be more coherent.
I think I'm starting to have a good global view and some ideas on what we can improve